src/app/workflow/launch-third-party/launch-third-party.component.html

```diff
@@ -1,22 +1,34 @@
-<div *ngIf="dnastackURL || fireCloudURL" class="panel panel-default">
+<div *ngIf="isWdl()" class="panel panel-default">
```

Comment:
Check how many times this is called.

Reply:
Made this a property instead of a function call in subsequent commit.